README.md

```diff
@@ -436,4 +437,23 @@ To order them:
       -c <cohort id> \
    ```
 
-3. Update `_config.yml` file to add new cohort in collection
\ No newline at end of file
+3. Update `_config.yml` file to add new cohort in collection
+
+## License
+
+Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)
```

Comment:
We should write more clearly what is going under which license.
Something like here: https://github.com/galaxyproject/training-material/#license